core/src/main/java/org/jdbi/v3/core/result/ResultIterable.java

```diff
@@ -76,6 +93,8 @@
     @Override
     ResultIterator<T> iterator();
 
+    StatementContext getContext();
```

Comment:
Does this refactor not work without this method? On the `ResultSetIterable` refactor, I took pains to avoid needing this method, so that it could be a `@FunctionalInterface`.